.github/workflows/cargo.yml

```diff
@@ -1,47 +1,99 @@
 on:
   push:
-    branches: [ master ]
```

Comment:
Why are we dropping the CI run for push to 'master' branch?

Reply:
We're not, now it just runs on pushes on all branches. I needed this for testing so that it would actually run in the CI for the fork branch.

I can put this back but I think that running CI on branches in forks is nicer than having to wait until a PR is opened.